### ISSUE TITLE
LWP::Protocol::https not specified as a prereq

### ISSUE THREAD

**Issue by eserte** [metacpan/metacpan-client#44]
Tests fail if LWP::Protocol::https is not installed:
```
Use of uninitialized value in concatenation (.) or string at /opt/perl-5.24.0t/lib/5.24.0/HTTP/Tiny.pm line 526.

#   Failed test 'No illegal methods called'
#   at t/ua_trap.t line 45.
#          got: 'Failed to fetch 'https://api.metacpan.org/v0/author/KENTNL': Protocol scheme 'https' is not supported (LWP::Protocol::https not installed) at /tmpfs/.cpan-build-cpansand/2016080518/MetaCPAN-Client-1.022001-AMgyxF/blib/lib/MetaCPAN/Client.pm line 201.
# '
#     expected: undef
# Failed to fetch 'https://api.metacpan.org/v0/author/KENTNL': Protocol scheme 'https' is not supported (LWP::Protocol::https not installed) at /tmpfs/.cpan-build-cpansand/2016080518/MetaCPAN-Client-1.022001-AMgyxF/blib/lib/MetaCPAN/Client.pm line 201.
# Looks like you failed 1 test of 1.
t/ua_trap.t .............. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

**Comment by mickeyn** [comment]
released `MetaCPAN::Client 1.022002` to fix this (hopefully)

**Comment by eserte** [comment]
I think this particular issue is fixed.
